.travis.yml

```diff
@@ -3,6 +3,9 @@ node_js:
   - 6
   - 8
   - 10
+script:
+  - npm run build
+  - npm run test
```

Comment:
what do these do? couldn't find it in travis docs

Reply:
normally travis just runs `npm run test` to check if everything works, but here the code must be transpiled first - it's not done by default, so it must be added manually
https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#default-build-script